### ISSUE TITLE
Unable to install PySAD in python 3.12 (ubuntu)

### ISSUE THREAD

**Issue by onixlas** [selimfirat/pysad#10]
PySAD installation in python 3.12 fails due to old NumPy and SciPy versions in requirements.txt.

Setting
```
numpy>=1.23.5
scipy>=1.10.0
```
fixes it.

OS: Ubuntu 24.10 amd64
Python 3.12.7

**Comment by dilky97** [comment]
Facing the same issue, how can I use this fix locally?

**Comment by onixlas** [comment]
> Facing the same issue, how can I use this fix locally?

Hello @dilky97 , you can install the fixed version directly from my fork using pip:

```bash
pip install git+https://github.com/onixlas/pysad.git
```

This will pull the latest code from my repository, which includes the fix for the issue. Once the PR is merged into the main repo, you can switch back to the official version.

Let me know if you run into any problems!

**Comment by dilky97** [comment]
Hi @onixlas!

Thank you for the quick response. I was able to install it using the link provided.

**Comment by selimfirat** [comment]
We've just published v0.3.0 thanks to @onixlas. You can directly use `pip install -U pysad` with python 3.12.
